tools/autograd/derivatives.yaml

```diff
@@ -305,7 +305,10 @@
   end: grad * weight
 
 - name: lgamma(Tensor self)
-  self: not_implemented("lgamma")
+  self: grad * digamma(self)
```

Comment:
Do I understand correctly that this is all that is needed to register `digamma` as the grad of `lgamma`?

Reply:
Yes, that's correct